modules/virt-define-http-liveness-probe.adoc

```diff
@@ -4,74 +4,41 @@
 
 [id="virt-define-http-liveness-probe_{context}"]
 
-= Define an HTTP liveness probe
+= Defining an HTTP liveness probe
+
+Define an HTTP liveness probe by setting the `spec.livenessProbe.httpGet` field of the virtual machine instance (VMI) configuration. Both HTTP and TCP tests for liveness probes can be defined in the same way as readiness probes. This procedure configures a sample liveness probe with an HTTP GET test.
```

Comment:
I might rephrase this sentence to the active voice: "Both HTTP and TCP tests for liveness probes can be defined in the same way as readiness probes."